src/media/js/routes_api.js

```diff
@@ -9,7 +9,7 @@ define('routes_api', [], function() {
         'fxa-login': '/api/v2/account/fxa-login/',
         'login': '/api/v2/account/login/',
         'logout': '/api/v2/account/logout/',
-        'search': '/api/v1/apps/search/suggest/non-public/?cache=1&vary=0',
+        'search': '/api/v1/apps/search/non-public/?cache=1&vary=0',
```

Comment:
I was thinking about lookup b/c of the suggest api but not sure anymore. I suppose it still could be. But even so the lookup needs to find deleted apps and this one doesn't so I'll likely just end up using the same API endpoint and requiring new permissions. But now with the search DRF filters it makes this super easy.